main.go

```diff
@@ -71,7 +85,11 @@ func main() {
 		return nil
 	}
 
-	app.Run(os.Args)
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Println(err.Error())
```

Comment:
If you are going to use the log package, I'd probably just go for `log.Fatal(err)`

Reply:
Ah fair, wouldn't want the error message getting discarded.